common/src/main/java/com/wynntils/screens/characterselector/widgets/DisconnectButton.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.characterselector.widgets;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.screens.characterselector.CharacterSelectorScreen;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.mc.TooltipUtils;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.screens.TitleScreen;
+import net.minecraft.client.gui.screens.inventory.tooltip.ClientTooltipComponent;
+import net.minecraft.network.chat.Component;
+
+public class DisconnectButton extends WynntilsButton {
+    private static final List<Component> TOOLTIP = List.of(
+            Component.translatable("screens.wynntils.characterSelection.disconnectButton.disconnect")
+                    .withStyle(ChatFormatting.GREEN),
+            Component.translatable("screens.wynntils.characterSelection.disconnectButton.description")
+                    .withStyle(ChatFormatting.GRAY));
+    private final CharacterSelectorScreen characterSelectorScreen;
+
+    public DisconnectButton(int x, int y, int width, int height, CharacterSelectorScreen characterSelectorScreen) {
+        super(x, y, width, height, Component.literal("Disconnect Button"));
+        this.characterSelectorScreen = characterSelectorScreen;
+    }
+
+    @Override
+    public void onPress() {
+        TitleScreen titleScreen = new TitleScreen();
```

Comment:
This looks like we are doing a lot of vanilla housekeeping here. It sounds scary, like it could break at any upgrade. There is no one single method that we can call? What does the vanilla disconnect button do? Can't we call that method instead?